### ISSUE TITLE
UI Component - Audio

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#13533]
This issue is to track the creation of the component: **Audio**

You can access the link at https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=688%3A42604.

Please have a chat with the design team before starting with the component, and don't forget to add sizing to the issue

**Comment by briansztamfater** [comment]
I think it is a duplicate of #13530

**Comment by cammellos** [comment]
My bad, thank you
